packages/core/content-type-builder/admin/src/components/AIChat/Chat.tsx

```diff
@@ -393,7 +397,8 @@ const Chat = () => {
     'Strapi AI can make mistakes.'
   );
 
-  if (!isChatEnabled) {
+  // Don't render the chat at all in production mode or if chat is not enabled
+  if (!isChatEnabled || isProduction) {
```

Comment:
This should be at the CTB level, since the chatbox will work in production everywhere else. If we just want this quick fix, it works for now, but we'll have to update it again in the next few weeks